sqlx-cli/src/bin/cargo-sqlx.rs

```diff
@@ -1,22 +1,19 @@
-use clap::{crate_version, AppSettings, FromArgMatches, IntoApp};
+use clap::Parser;
 use console::style;
 use dotenv::dotenv;
 use sqlx_cli::Opt;
-use std::{env, process};
+use std::process;
+
+#[derive(Parser, Debug)]
+#[clap(bin_name = "cargo")]
+enum Cli {
+    Sqlx(Opt)
+}
```

Comment:
This is a very cool hack, although would you mind adding a comment here explaining why this is necessary? Similar to the comment that was previously at line 9

Reply:
Added 😄